.docker/Dockerfile.php

```diff
@@ -1,3 +1,11 @@
+##
+# `php` image
+#
+# This image is derived from php-fpm, which lacks tools to build Drupal.
+# The files are therefore copied from the `cli` container (Dockerfile.cli) which is built first.
+#
+
+# CLI_IMAGE set in &lagoon-project in docker-compose.yml
 ARG CLI_IMAGE
 FROM ${CLI_IMAGE} as cli
 
```

Comment:
I'm wondering whether having PaaS have a hard dependency on SaaS here is wise - maybe we do as we did for cli and go `FROM amazeeio/php:7.2-fpm-latest` - but comment that they should refer to our govcms8lagoon dockerfile for a best-practice setup (we only add clam - for now - and sanitise in SaaS)?

Reply:
That's how we were expecting to use it.

Reply:
And clam will be removed from here, as I understand it.